lib/marked.js

```diff
@@ -664,7 +664,7 @@ InlineLexer.prototype.output = function(src) {
     // code
     if (cap = this.rules.code.exec(src)) {
       src = src.substring(cap[0].length);
-      out += this.renderer.codespan(escape(cap[2].trim(), true));
+      out += this.renderer.codespan(escape(cap[3].trim(), true));
```

Comment:
@Feder1co5oave @joshbruce We have to watch out for changing capturing parentheses. This can be a breaking change. https://github.com/firede/msee/pull/59/files